test/node.js

```diff
@@ -57,42 +59,44 @@ test('node pass on nodeOptions', async t => {
 test.serial(
 	'node removes --inspect from nodeOptions when defined by parent process',
 	inspectMacro,
-	'--inspect'
+	'--inspect',
 );
 
 test.serial(
 	'node removes --inspect=9222 from nodeOptions when defined by parent process',
 	inspectMacro,
-	'--inspect=9222'
+	'--inspect=9222',
 );
 
 test.serial(
 	'node removes --inspect-brk from nodeOptions when defined by parent process',
 	inspectMacro,
-	'--inspect-brk'
+	'--inspect-brk',
 );
 
 test.serial(
 	'node removes --inspect-brk=9222 from nodeOptions when defined by parent process',
 	inspectMacro,
-	'--inspect-brk=9222'
+	'--inspect-brk=9222',
 );
 
 test.serial(
 	'node should not remove --inspect when passed through nodeOptions',
 	async t => {
 		const {stdout, stderr} = await execa.node('console.log("foo")', {
 			reject: false,
-			nodeOptions: ['--inspect', '-e']
+			nodeOptions: ['--inspect', '-e'],
 		});
 
 		t.is(stdout, 'foo');
 		t.true(stderr.includes('Debugger listening'));
-	}
+	},
 );
 
 test('node\'s forked script has a communication channel', async t => {
-	const subprocess = execa.node('test/fixtures/send');
+	const subprocess = execa.node('test/fixtures/send.js');
+	await pEvent(subprocess, 'message');
```

Comment:
Without waiting for a [message](https://github.com/sindresorhus/execa/pull/478/files#diff-26a0afea745973df4bc95c26bdead61741221bdd9952dbf06540d5f69178ceec) from `send.js`, it seems like `subprocess.send('ping')` would happen before it gets a chance to install a listener. Possibly because ES modules are async?

Reply:
Interesting. Your explanation is a possible cause.
Regardless, the core of the test, i.e. testing the forked script's communication channel is kept, so this looks good to me. Thanks for fixing it :+1: